ethereum/blockcreation/src/test/java/org/hyperledger/besu/ethereum/blockcreation/PoWBlockCreatorTest.java

```diff
@@ -52,9 +52,9 @@
 
 import com.google.common.collect.Lists;
 import org.apache.tuweni.bytes.Bytes;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-public class PoWBlockCreatorTest {
+public class PoWBlockCreatorTest extends AbstractBlockCreatorTest {
```

Comment:
Do you think we need a MergeBlockCreatorTest as well?

Reply:
We should have one, the only thing different about the MergeBlockCreator is the block header it creates, but it would be good to have a test for that. Do it this PR or another?